recipes/dbt-duckdb/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "dbt-duckdb" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dbt-duckdb-{{ version }}.tar.gz
+  sha256: e36776d6ee7aa1554a236b7ffa93c556c4419920134a3801feef7c64b706db16
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - dbt-core >=1.6.0,<1.7.dev0
```

Keep it simple. Dev versions won't be on the main channel anyways.
```suggestion
    - dbt-core >=1.6.0,<1.7
```